roles/foreman/tasks/main.yaml

```
@@ -46,6 +46,11 @@
     name: foreman-client-key
     path: "{{ foreman_client_key }}"
 
+- name: Deploy Foreman socket
+  ansible.builtin.template:
+    src: foreman.socket.j2
+    dest: /etc/systemd/system/foreman.socket
```

```suggestion
    dest: /etc/systemd/system/foreman.socket
    mode: '0644'
```
to make Ansible happy